### DISCUSSION TITLE
Simple Armory Performance Improvement

### DISCUSSION THREAD

**Discussion by eTzmNcbkrng** [chromiecraft/chromiecraft#3400] [Suggestions]
As much as I like the Animated 3D Cata models in the Armory, it really hammers my CPU when looking at characters, could this be made into an optional feature either account-wide or a toggle after opening any given armory page.

Characters' gear loadout always loads quickly enough, but my PC always sounds like it's about to send me into orbit until the animated models have finished loading, which can take minutes sometimes.

Edit: I've managed to significantly reduce load times by disabling the models with uBlock Origin.
In uBlock Origin settings, go to the My Filters page and add this: 
```
! https://www.chromiecraft.com
www.chromiecraft.com###model
```
Then click Apply Changes

**Comment by r-o-b-o-t-o** [comment]
Hi there,
This is being worked on here https://github.com/r-o-b-o-t-o/azerothcore-armory/pull/17 and might be pushed to ChromieCraft in the following days.

Best regards,
Roboto
